src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/GetPrintableDocumentsForEvent.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object GetPrintableDocumentsForEvent extends PerformanceTestsConfig {
+
+  private val url: String = config.getString("caseDataUrl") + "/" + config.getString("getPrintableDocumentsForEvent")
+  println("Retrieving getPrintableDocumentsForEvent URL : " + url)
+
+  def httpRequest() = {
+    val s2sToken = CcdTokenGenerator.generateGatewayS2SToken()
+    val userToken = CcdTokenGenerator.generateWebUserToken()
+    http("TX09_CCD_GetPrintableDocumentsForEvent_getDocumentsForEvent")
```

Comment:
what's the meaning of TX09 ?

Reply:
that's transaction name needed to identify timing for specific endpoint from gatling raw data